code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const ThoughtForm = ({ handleFormSubmit, onNewThoughtChange, newThought }) => {
+  return (
+  <section className="form-section">
+    <form onSubmit={handleFormSubmit}>
+      
+      <label htmlFor="newThought">
+      <p>What's making you happy now?</p></label>
+      <textarea
```

Comment:
Add "maxLength='140'" to the textarea so that the number of characters can't exceed 140?